lib/types/types.ts

```diff
@@ -48,6 +48,8 @@ export type applicantType = {
   periodId: string | ObjectId;
 };
 
+export type bankomOptionsType = "yes" | "no" | "maybe" | undefined;
```

Comment:
tror vi kan bruke denne mange andre steder, men kan sikkert fikse det en annen gang.

ser også egt ikke noe grunn til å ha denne typen på engelsk, blir jo bare ett ekstra ledd for å oversette når den skal vises. regner med at det er brukt såpass mange steder, at vi kan fikse det en annen gang.

Reply:
Ja, blir litt hassel å oversette det, men kan sette opp det samt standarisere som et eget issue